src/topology/metric_space/basic.lean

```diff
@@ -779,6 +779,15 @@ lemma cauchy_seq_iff_tendsto_dist_at_top_0 [nonempty β] [semilattice_sup β] {u
 by rw [cauchy_seq_iff_tendsto, metric.uniformity_eq_comap_nhds_zero, tendsto_comap_iff,
   prod.map_def]
 
+lemma tendsto_uniformity_iff_dist_tendsto_zero {ι : Type*} {f : ι → α × α} {p : filter ι} :
+  tendsto f p (𝓤 α) ↔ tendsto (λ x, dist (f x).1 (f x).2) p (𝓝 0) :=
+by rw [metric.uniformity_eq_comap_nhds_zero, tendsto_comap_iff]
+
+lemma filter.tendsto.congr_dist {ι : Type*} {f₁ f₂ : ι → α} {p : filter ι} {a : α}
```

Comment:
I don't think I would find this name. But the more descriptive alternatives would be much longer I guess.

Reply:
We can have an alias. What name would you suggest? I'd like to have a version in the `filter.tendsto` namespace.

Reply:
@PatrickMassot ping here

Reply:
Sorry, I forgot about this conversation. Maybe `filter.tendsto_of_tendsto_of_dist`? But we can also keep your name only.

Reply:
Added an alias `tendsto_of_tendsto_of_dist` (without `filter.`).